BasicSample/Assets/Editor/MixedRealitySampleQuickSetupWindow.cs

```diff
@@ -104,17 +104,23 @@ private void OnGUI()
             m_selectedMRConfiguration = MixedRealityProjectConfiguration.None;
             EditorGUIUtility.labelWidth = Default_Label_Width;
             GUIStyle titleStyle = new GUIStyle(EditorStyles.largeLabel) { fontStyle = FontStyle.Bold };
-            GUIStyle contentStyle = new GUIStyle(EditorStyles.label) { fontStyle = FontStyle.Bold };
+            GUIStyle headingStyle = new GUIStyle(EditorStyles.label) { fontStyle = FontStyle.Bold };
+            GUIStyle contentStyle = new GUIStyle(GUI.skin.box);
+            if(EditorGUIUtility.isProSkin)
```

Comment:
nit: now that the textColor is coming from the EditorStyles, is there still a need for this `if(EditorGUIUtility.isProSkin)` check? Shouldn't this be fine to run always?